src/main/java/com/zebrunner/mcloud/grid/MobileRemoteProxy.java

```diff
@@ -78,6 +82,24 @@ protected Boolean initialize() throws ConcurrentException {
             return true;
         }
     };
+    public static final Map<String, Duration> DEVICE_IGNORE_AUTOMATION_TIMERS = new ConcurrentHashMap<>();
+
+    private static final Duration DOWN_TIMEOUT = Optional.ofNullable(System.getenv("DOWN_TIMEOUT"))
+            .filter(StringUtils::isNotBlank)
+            .map(Integer::parseInt)
+            .map(Duration::ofSeconds)
+            .orElse(Duration.ofMinutes(3));
+    private static final Duration UNHEALTHY_WDA_ADB_TIMEOUT = Optional.ofNullable(System.getenv("UNHEALTHY_WDA_ADB_TIMEOUT"))
```

Comment:
let's rename to UNHEALTHY_MOBILE_TIMEOUT and just a comment regarding adb and wda

Reply:
done